packages/stores/src/account/base.ts

```diff
@@ -1185,10 +1192,14 @@ export class AccountStore<Injects extends Record<string, any>[] = []> {
     gasLimit,
     chainId,
     address,
+    checkOtherFeeTokens = true,
```

Comment:
Is there a reason we wouldn't want the behavior by default?

Reply:
This is used for 1-Click trading. I accidentally included this from that PR. I'll remove this as we don't need it now